stages/org.osbuild.oscap.autotailor

```diff
@@ -6,6 +6,9 @@ The autotailor stage produces a tailoring file that the OpenSCAP scanner can use
 and remediate a system. The autotailor rules override a base profile either enabling or
 disabling (selecting or unselecting) a given rule for the profile. The autotailor command
 generates and xml diff between the user provided overrides and the base profile.
+
+This stage depends on the `autotailor` binary being available in the build root,
```

Comment:
Small wording suggestion -- it looks like we've used a 'Notes:' section of the docstring to list things like this. See the top of `org.osbuild.authconfig` for an example. So far it looks like authconfig, authselect, and chown follow this convention.

Reply:
Cool, thanks. I'll take a look